src/index.ts

```diff
@@ -1,14 +1,29 @@
 import { setFailed, setOutput } from "@actions/core";
 import { Octokit } from "@octokit/action";
+import { OctokitOptions } from "@octokit/core/dist-types/types";
 import { getInputs } from "./getInputs";
+import { HttpsProxyAgent } from "https-proxy-agent";
 
 async function run(): Promise<void> {
   try {
-    const { reviewers, ...pullParams } = getInputs();
+    const { reviewers, baseurl, ...pullParams } = getInputs();
 
-    const octokit = new Octokit();
+    const options: OctokitOptions = {};
+    if (baseurl !== undefined) {
+      options.baseUrl = `${baseurl}/api/v3`;
+    }
+
+    const proxy = process.env.https_proxy || process.env.HTTPS_PROXY;
```

Comment:
Is this some kind of standardized env variable? I think I'd prefer this to be a input because it is easier to test.

Reply:
yes it is standardized env variable. i have seen various other github actions doing very similar.

Reply:
Okay, can we document this in the README then please?

Reply:
sure will do.